tests/readme.md

```diff
@@ -0,0 +1,16 @@
+# Tests
+
+We have split the test suite into two sections
+- unittests: low level tests
```

Comment:
The reason I went with `pytest` "marks" instead of an unittest / integration split is because we could have integration tests that don't require an external API (of unit tests calling the external API). Actually the last sentence in this file implies that unit tests are calling the API!

Not a strong opinion so I'll let you make the call. If you keep the unittest/integration split, you'll need to update the main README.